src/mon/MonCommands.h

```diff
@@ -1378,8 +1378,25 @@ COMMAND("config generate-minimal-conf",
 	"Generate a minimal ceph.conf file",
 	"config", "r")
 
+/* NVMeofGwMon*/
+COMMAND("nvme-gw create"
```

Comment:
Commands would need to be aware about the configurable as well.

Reply:
Agree!
This is an intrusive change and should be done separately not when adding a new service.

Reply:
The cephadm is the only one calling create/delete when deploying the nvmeof service. If the nvmeof service is not being deployed, these commands are not called at all. If the nvmeof service is being deployed, these commands must be called. If we want to add another safety mechanism like a config option to verify that it is not being called by mistake, it is possible, but I don't think that it should hold off this PR.

Reply:
> This is an intrusive change and should be done separately not when adding a new service.

I'm afraid the new service is far more intrusive to the monitor than the configurable to the commands & `paxos_service`.